profiles/coreos/base/package.accept_keywords

```diff
@@ -115,3 +115,5 @@ dev-util/checkbashisms
 # fix for https://github.com/coreos/bugs/issues/324
 =net-misc/curl-7.41
 
+# Only needed until go-1.5 is available upstream
+dev-lang/go-bootstrap ~amd64
```

Comment:
Might as well just set KEYWORDS=amd64 in the ebuild since it is here in the overlay.